app/client/packages/design-system/widgets/src/components/Datepicker/src/DatepickerTrigger.tsx

```
@@ -0,0 +1,49 @@
+
+interface DatepickerTriggerProps {
+  isLoading?: boolean;
+  size?: Omit<keyof typeof SIZES, "xSmall" | "large">;
+  isDisabled?: boolean;
+}
+
+export const DatepickerTrigger = (props: DatepickerTriggerProps) => {
+  const { isDisabled, isLoading, size } = props;
+
+  const suffix = useMemo(() => {
+    if (Boolean(isLoading)) return <Spinner />;
```

Biome

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)